admin-dev/themes/new-theme/scss/components/_grid.scss

```diff
@@ -309,10 +309,6 @@ table {
 
       td[data-column-id^="id_"] {
         max-width: 8rem;
-
-        input {
```

Comment:
Are you sure about this change?
As I see the id column size was made smaller intentionally by @NeOMakinG in https://github.com/PrestaShop/PrestaShop/pull/24952

Reply:
Yes. The size of columns will be worked on a new PR.